sotodlib/tod_ops/sss.py

```diff
@@ -0,0 +1,216 @@
+import numpy as np
```

Comment:
Add comment at the top about what this module is for.  Specifically State Something about what SSS stands for, cause that's not obvious to everyone.

I've seen SSS most often on my boarding pass, and it means I'm getting extra security checks.

Reply:
Yeah, let's change it to azss (Azimuth synchronous signal).

BTW, there are 253 definitions for 'SSS', lol (https://www.abbreviations.com/SSS)

Reply:
Replaced all SSS I found with AzSS.